docs/self-hosted/kubernetes-migration-guide.md

```diff
@@ -0,0 +1,144 @@
+
+## Restore application data in Kubernetes
+
+All commands assume that you are logged in to the server where Private Packagist Self-Hosted Kubernetes is installed,
+your working directory contains the three files generated by the previous steps, and that
+Private Packagist Self-Hosted Kubernetes is installed in the Kubernetes namespace "default".
+
+To avoid any data inconsistency, we highly recommend going through the steps below in order and only restart the application
+once all the data has been restored.
+
+### Stop the Private Packagist application
+
+```
+kubectl scale deployment ui repo worker --replicas 0 -n default
+```
+
+### Restore the PostgreSQL database
+
+Follow the instructions below to restore the PostgreSQL database in the cluster.
+If you are using your own PostgreSQL instance, use the `packagist_db.sql` file directly to import the data into your instance.
+
+Please note that the backup will overwrite any existing data in the database and any changes you made during setup will be lost.
+
+```
+kubectl scale statefulset postgres --replicas 0 -n default
+kubectl delete pvc postgres-data-postgres-0 -n default
+kubectl scale statefulset postgres --replicas 1 -n default
+cat packagist_db.sql | kubectl exec -it -n default statefulsets/postgres -- psql -U postgres -d packagist_db
+```
+
+### Restore the Redis database
+
+Follow the instructions below to restore the Redis database in the cluster.
+If you are using your own Redis instance, use the `packagist_redis.rdb` file directly to [restore Redis from an RDB file](https://redis.io/learn/guides/import#restore-an-rdb-file) or follow the instructions of your hosted service provider.
+
+Follow these steps if you are using Private Packagist Self-Hosted with an embeded Kubernetes cluster
+```
+export PV_NAME=$(kubectl get pvc -n default redis-data-redis-0 -ojsonpath='{.spec.volumeName}')
+kubectl scale statefulset -n default redis --replicas 0
+rm /var/openebs/local/$PV_NAME/appendonly.aof
+cp packagist_redis.rdb /var/openebs/local/$PV_NAME/dump.rdb
+kubectl scale statefulset -n default redis --replicas 1
+```
+
+### Restore the dist and artifact files
+
+Please note that depending on the size of the `packagist_storage.tar.gz` file and the type of blob storage that you have configured
+it can take several minutes for this command to finish.
+
+```
+kubectl scale statefulset -n default ui --replicas 1
+kubectl cp -n default packagist_storage.tar.gz ui:/tmp/packagist_storage.tar.gz
+kubectl exec -n default ui /srv/manager/bin/console packagist:self-hosted:migrate-storage import /tmp/packagist_storage.tar.gz && rm /tmp/packagist_storage.tar.gz 
+```
+
+### Start the Private Packagist application
+
+Once your data has been restored. Restart the application, log in to Private Packagist, and verify that your organization
+and packages are there again.
+
+```
+kubectl scale deployment ui --replicas 1 -n default
+```
```

Comment:
```suggestion
kubectl scale deployment ui --replicas 1 -n default
```

Log in to (the new?) Private Packagist, and verify that your organization and packages are there again.
```